modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/cloud/BlobStorage.scala

```diff
@@ -35,6 +35,8 @@ trait BlobStorage[F[_]] {
 
   def get(path: BlobStorage.Key): F[Either[Throwable, String]]
 
+  def getBytes(path: BlobStorage.Key): Stream[F, Byte]
```

Comment:
Could you put the implementation of `get` here in this trait?  Instead of repeating it in each of AWS/GCS/Blob.  I believe the implementation of `get` is the same everywhere; it's only the implementation of `getBytes` which changes.

Reply:
If we put `get` implementation in this trait: 

```scala
getBytes(key).compile
   .to(Array)
   .map(array => new String(array, StandardCharsets.UTF_8))
   .attempt
```

then it means we have to also put some constraints for `F` here, like `Sync`? It would be problematic for pure test implementation of blob storage?